src/sds.c

```diff
@@ -454,7 +454,7 @@ sds_2byte_read (SF_PRIVATE *psf, SDS_PRIVATE *psds)
 
 	ucptr = psds->read_data + 5 ;
 	for (k = 0 ; k < 120 ; k += 2)
-	{	sample = arith_shift_left (ucptr [k], 25) + arith_shift_left (ucptr [k + 1], 18) ;
+	{	sample = arith_shift_left (ucptr [k], 25) | arith_shift_left (ucptr [k + 1], 18) ;
```

Comment:
@ codeowners
Based on my reading of what the sampling loop is trying to do. I *think* a bitwise OR is really what should have been done in the first place. Arithmetically, these should be equivalent, **unless** the ucptr bytes are supposed to be interpreted as signed values. I don't know why they would be, but I'll stand to be wrong.

Reply:
>Can you include identical changes to sds_3byte_read() and sds_4byte_read() please.

Will do.

Reply:
Fixed patch V2.